.github/workflows/generate-summary.yaml

```diff
@@ -409,7 +409,7 @@ jobs:
       - name: Trim issue length # reduce the number of lines in final issue so github always creates issue
         run: |
           head -c 65000 issue.md > trimmed_issue.md
-          if [ $(cat trimmed_issue.md | wc -l) -ne $(cat issue.md | wc -l) ]; then printf "\n\`\`\`\nIssue text has been trimmed. Please check logs for the untrimmed issue.\n" >> trimmed_issue.md; fi
+          if [ $(stat -c%s issue.md) -gt 65000 ]; then printf "\nIssue text has been trimmed. Please check logs for the untrimmed issue.\n" >> trimmed_issue.md; fi
```

Comment:
I think this one in particular needs to have the \n\`\`\` for some reason but the others don't

Reply:
did it fail before? I can wait until staging is done and see what happens without the \n```

Reply:
I vaguely remember it failing before though I can't remember if it was just a result of using 
```
echo "\n\`\`\`
```